src/Components/v2/ArtworkFilter/ArtworkFilterArtworkGrid2.tsx

```diff
@@ -0,0 +1,158 @@
+import { Box, Spacer } from "@artsy/palette"
+import { ArtworkFilterArtworkGrid2_filtered_artworks } from "__generated__/ArtworkFilterArtworkGrid2_filtered_artworks.graphql"
+import { AnalyticsSchema, useSystemContext, useTracking } from "Artsy"
+import ArtworkGrid from "Components/ArtworkGrid"
+import React, { useEffect, useState } from "react"
+import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay"
+import { LoadingArea } from "../LoadingArea"
+import { PaginationFragmentContainer as Pagination } from "../Pagination"
+import { useFilterContext } from "./ArtworkFilterContext"
+import { ArtworkFilterZeroState } from "./ArtworkFilterZeroState"
+
+interface ArtworkFilterArtworkGridProps {
+  columnCount: number[]
+  filtered_artworks: ArtworkFilterArtworkGrid2_filtered_artworks
```

Comment:
We need to fix a naming collision inside the `/artist` app by renaming that component to `ArtistArtworkFilterArtworkGrid`.